libs/utils/src/ballot_styles.ts

```diff
@@ -78,16 +73,19 @@ export function getRelatedBallotStyle(params: {
   }
 
   // For legacy language-agnostic ballot styles, return the same ballot style:
-  if (_.isEmpty(sourceBallotStyle.languages)) {
+  if (
+    !sourceBallotStyle.languages ||
+    sourceBallotStyle.languages.length === 0
```

Comment:
I admit this would be nicer to have a helper for. Maybe if `iter` could accept an optional value and treat it like `[]`, then add a `isEmpty` method?

```ts
if (iter(sourceBallotStyle.languages).isEmpty()) {
```

Reply:
Sure, that could work

Reply:
https://github.com/votingworks/vxsuite/pull/4426